verifier/src/main/kotlin/tech/figure/classification/asset/verifier/config/VerifierClientConfig.kt

```diff
@@ -160,6 +165,7 @@ class VerifierClientConfig private constructor(
             eventDelegator = eventDelegator ?: AssetClassificationEventDelegator.default(),
             eventProcessors = eventProcessors,
             okHttpClientBuilder = okHttpClientBuilder ?: { defaultOkHttpClient() },
+            eventStreamProvider = eventStreamProvider ?: DefaultEventStreamProvider(eventStreamNode ?: URI("ws://localhost:26657"), okHttpClientBuilder?.invoke() ?: defaultOkHttpClient())
```

Comment:
I've gone ahead and removed those from the config itself, but left them as a part of the builder. I wasn't sure how you wanted to deal with the case a consumer wanted to use the default provider with a custom URI / http client. Thoughts?

Reply:
Ohhhh, I was thinking you'd actually add those values to the `DefaultEventStreamProvider` and make them non-null, like:
```
class DefaultEventStreamProvider(
   eventStreamNode: URI = URI("ws://localhost:26657"),
   httpClient: OkHttpClient = defaultOkHttpClient()
)
```

Then, when using the builder, you could customize one or both in its declaration:
```
val config = VerifierClientConfig.builder(x, y, z).withEventStreamProvider(DefaultEventStreamProvider(eventStreamNode = myCustomNode, httpClient = myCustomClient))
```

Which would allow the defaults to be used if desired, or allow one or both to be modified.

Reply:
Mostly saying this because `eventStreamNode` and `httpClient` should probably be omitted from the configs.  Once the Block API stuff is added, those values will probably only pertain to the event stream code we currently have and maybe won't make sense if the event stream provider is block api-related, or a custom implementation

Reply:
ah, gotcha. makes sense. done and done :D